src/picongpu/include/particles/bremsstrahlung/SynchrotronFunctions.hpp

```diff
@@ -22,12 +22,10 @@
 
 #include "pmacc_types.hpp"
```

Comment:
please inlcude `simulation_defines.hpp` to get the definition of `float_X`.
"pmacc_types.hpp" can be removed.